src/main/java/hudson/remoting/Engine.java

```diff
@@ -664,7 +664,14 @@ public void afterChannel(@Nonnull JnlpConnectionState event) {
                 // try to connect back to the server every 10 secs.
                 resolver.waitForReady();
 
-                events.onReconnect();
+                try {
+                    events.status("Performing onReconnect operation.");
```

Comment:
How often is this operation performed? My concern is the user gets flooded by logs.

Reply:
In theory it never occurs because the network, system, and environment are reliable and well-behaved. Reality is a bit different, but it should be a fairly rare. I can see it happen when my Windows machine goes to sleep but otherwise I never see it in my environment. Others report it more frequently.

This change keeps the frequency but reduces and the actual message.

Reply:
Fair enough, if it's not something that should happen often, it makes sense to me.